pkg/cli/root.go

```diff
@@ -79,6 +82,46 @@ func NewRootCmd() *cobra.Command {
 		},
 	}
 
+	// "scrub"
+	scrubCmd := &cobra.Command{
+		Use:     "scrub <config>",
+		Aliases: []string{"scrub"},
+		Short:   "`scrub` checks manifest/blob integrity",
+		Long:    "`scrub` checks manifest/blob integrity",
+		Run: func(cmd *cobra.Command, args []string) {
+			configuration := config.New()
+
+			if len(args) > 0 {
+				LoadConfiguration(configuration, args[0])
+			}
+
+			ln, err := net.Listen("tcp", fmt.Sprintf("%s:%s", configuration.HTTP.Address, configuration.HTTP.Port))
+
+			defer func(ln net.Listener) {
+				err = ln.Close()
+				if err != nil {
+					panic(err)
+				}
+			}(ln)
+
+			if err != nil {
+				fmt.Println("Server is running")
```

Comment:
Can we use Info log here instead of println ?

Reply:
Done